rotkehlchen/chain/evm/decoding/curve/constants.py

```diff
@@ -27,26 +28,29 @@
 GAUGE_VOTE: Final = b'E\xca\x9aL\x8d\x01\x19\xeb2\x9eX\r(\xfeh\x9eHN\x1b\xe20\xda\x807\xad\xe9T}-%\xcc\x91'  # noqa: E501
 TOKEN_EXCHANGE: Final = b'\x8b>\x96\xf2\xb8\x89\xfaw\x1cS\xc9\x81\xb4\r\xaf\x00_c\xf67\xf1\x86\x9fppR\xd1Z=\xd9q@'  # noqa: E501
 TOKEN_EXCHANGE_UNDERLYING: Final = b'\xd0\x13\xca#\xe7ze\x00<,e\x9cTB\xc0\x0c\x80Sq\xb7\xfc\x1e\xbdL lA\xd1Sk\xd9\x0b'  # noqa: E501
+TOKEN_EXCHANGE_NG: Final = b'\x14?\x1f\x8e\x86\x1f\xbd\xed\xdd[F\xe8D\xb7\xd3\xac{\x86\xa1"\xf3n\x8cF8Y\xeeh\x11\xb1\xf2\x9c'  # noqa: E501
```

Comment:
What are these 2? What is `NG`? CAn you add a comment explaining?

Reply:
NG means new generation, we include these new contracts' topics to decode the events done through new contracts